indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -697,50 +697,70 @@
   (fn [_context concept-id _revision-id _options]
     (cs/concept-id->type concept-id)))
 
-(defmethod delete-concept :default
-  [context concept-id revision-id options]
-  ;; Assuming ingest will pass enough info for deletion
-  ;; We should avoid making calls to metadata db to get the necessary info if possible
+(defn- delete-concept-default-helper
```

Comment:
None of these functions touched in this change have documentation. I don't remember if the linter requires private functions to be documented, but the functions intent is good to capture now while you remember.